manifests/params.pp

```diff
@@ -791,4 +791,10 @@
   } else {
     $verify_command = '/usr/sbin/apachectl -t'
   }
+
+  if ($::osfamily == 'RedHat' and ($facts['operatingsystemmajrelease'] >= '8') ) {
```

Comment:
since Puppet deprecated those facts, please use the new ones:
```suggestion
  if ($facts['os']['family'] == 'RedHat' and ($facts['os']['release']['major'] >= '8') ) {
```

Reply:
Updated  :)

(Was keeping it the same as the other fact uses in the params file -- they all need switching to the facts array really...)